man/systemd.network.xml

```diff
@@ -1721,6 +1721,17 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>MulticastToUnicast=</varname></term>
+          <listitem>
+            <para>Takes a boolean. Controls whether the bridge multicast to unicast works on top
+            of the multicast snooping feature of the bridge. Which means unicast copies are only
```

Comment:
hmm, this needs better wording. "the bridge multicast to unicast" is something that doesn't make sense to me. @keszybz any suggestion how to make this better?